tests/pipelines/stable_diffusion/test_stable_diffusion.py

```diff
@@ -31,13 +31,16 @@
     VQModel,
 )
 from diffusers.utils import floats_tensor, load_image, slow, torch_device
+from diffusers.utils.testing_utils import require_torch_gpu
 from transformers import CLIPTextConfig, CLIPTextModel, CLIPTokenizer
 
+from ...test_pipelines_common import PipelineTesterMixin
+
 
 torch.backends.cuda.matmul.allow_tf32 = False
 
 
-class PipelineFastTests(unittest.TestCase):
+class StableDiffusionPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
```

Comment:
Making the class name a bit more verbose here, for easier search withing the test logs